numeric.c

```diff
@@ -3930,6 +3930,22 @@ fix_div(VALUE x, VALUE y)
     return fix_divide(x, y, '/');
 }
 
+/*
+ * call-seq:
+ *   self / numeric -> numeric_result
+ *
+ * Performs division:
+ *
```

Comment:
I know it wasn't in the original documentation, but I really think it is worth it to document that integer division truncates if the divisor is integer.  The examples show this, but I think it is important enough to mention in the description.

Reply:
I've already begun changes for the next PR.  Ok to add this then?

Reply:
Sure, that's fine.